README.md

```diff
@@ -60,17 +60,24 @@ There are at least four ways you can use SigMF today, thanks to the community-su
 
 ## Contributing
 
-The SigMF standards effort is organized entirely within this Github repository.
+The SigMF standards effort is organized within this Github repository.
```

Comment:
I would prefer if we qualify this with `core SigMF standards effort` as opposed to removing the specific language here.

> The core SigMF standards effort is organized entirely within this Github repository.

Reply:
I recommend using a word other than "core" since we tend to overload that term in the SigMF context, but agreed.